conda-forge.yml

```
@@ -4,12 +4,14 @@ os_version:
   linux_aarch64: cos7
   linux_ppc64le: cos7
 provider:
-  linux_ppc64le: azure
   linux_aarch64: azure
+  linux_ppc64le: azure
```

Preserving original order

```suggestion
  linux_ppc64le: azure
  linux_aarch64: azure
```

---

Just curious, does this make a difference in practice or was it just to avoid unnecessary changes in the diff?